ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/backwardsync/BackwardSyncContext.java

```diff
@@ -101,17 +100,21 @@ public synchronized boolean isSyncing() {
         .orElse(Boolean.FALSE);
   }
 
-  public synchronized void updateHead(final Hash headHash) {
-    if (Hash.ZERO.equals(headHash)) {
-      maybeHead = Optional.empty();
-    } else {
-      maybeHead = Optional.of(headHash);
+  public synchronized void maybeUpdateTargetHeight(final Hash headHash) {
```

Comment:
Rename method to reflect what is still does after removing maybeHead

Reply:
If I am not wrong this is redundant, and the method can be removed, since the update of the target height is done in the `syncBackwardsUntil` method, that is called just after

Reply:
I think I will consider removing this and `possiblyMoveHead` in a subsequent PR. I want to study the code a bit more and don't want to invalidate this PR's testing.